pkg/repositories/transformers/execution.go

```diff
@@ -135,6 +136,10 @@ func UpdateExecutionModelState(
 				},
 			},
 		}
+	} else if request.Event.GetError() != nil {
```

Comment:
isn't this condition checked in line 145 below? can you merge the blocks?

Reply:
done